hazelcast/src/main/java/com/hazelcast/cache/impl/operation/CacheClearOperation.java

```diff
@@ -77,9 +77,9 @@ public void afterRun() throws Exception {
         IPartitionService partitionService = getNodeEngine().getPartitionService();
         if (partitionService.getPartitionId(name) == partitionId) {
             cacheService.sendInvalidationEvent(name, null, SOURCE_NOT_AVAILABLE);
+        } else {
```

Comment:
Why not increment the sequence on the partition owner?

Reply:
It is incremented behind the scenes in sendInvalidationEvent method..

Reply:
Ok, thanks!